rust/rebase/README.md

```diff
@@ -1,121 +1,30 @@
 # Rebase
 
-Rebase offers an extensible, opinionated, trait-based approach to building credentialing systems on top of the foundation of [ssi](https://github.com/spruceid/ssi).
+Rebase is a library for building credentialing witnesses and clients to interact with them. Rebase is built on top of [SSI](https://github.com/spruceid/ssi) and uses [DID](https://www.w3.org/TR/did-core/)s and [Verifiable Credentials (VCs)](https://www.w3.org/TR/vc-data-model/) to create portable, specified credentials. Rebase provides abstractions for working with witnessed cryptographic signature-based attestations, linking cryptographic keys to traditional profiles and identities, and working with delegations to improve the experience for developers and users of applications with user-owned data models, along with other libraries available from [Spruce](), like [Kepler]() for storage.
```

Comment:
For [Spruce], let's link to our GH for now, [Kepler] we can unlink at the moment.